test/cli_container_cp_test.go

```diff
@@ -13,7 +13,7 @@ import (
 	"github.com/gotestyourself/gotestyourself/icmd"
 )
 
-// APIContainerCopySuite is the test suite for container cp CLI.
+// PouchContainerCopySuite is the test suite for container cp CLI.
```

Comment:
good catch!

Reply:
:smile: Can you please review this PR again?